cypress/README.md

```diff
@@ -1,4 +1,30 @@
-# Cypress Cucumber tests for Carbon storybook
+# Testing Strategy
+
+## Unit Testing
+Unit tests are writing / maintenancing by the developers
+
+## Manual Testing
+### Components:
+1. Every component should be fully tested using functional testing / UI testing / Internationalisation testing
+2. Should be tested all stories for dedicated component (default / classic / multiple / validation / themes etc.), notes section and accessibility section.
```

Comment:
Please, remove the blank space, it'll look better :)

Reply:
fixed